mini-cap/src/pages/AddCondoForm.jsx

```diff
@@ -22,6 +22,8 @@ const AddCondoForm = () => {
       unitSize: "",
       parkingNumber: "",
       lockerNumber: "",
+        occupant: "",
```

Comment:
adding these fields to the condo makes it so they are being checked if empty in the handleSumbit (around line 81), so it always fails the check. we should add the fields after the check (occupant: "", status: "vacant")